integration-tests/kyuubi-kubernetes-it/src/test/scala/org/apache/kyuubi/kubernetes/test/deployment/KyuubiOnKubernetesTestsSuite.scala

```diff
@@ -15,13 +15,23 @@
  * limitations under the License.
  */
 
-package org.apache.kyuubi.kubernetes.test
+package org.apache.kyuubi.kubernetes.test.deployment
 
 import org.apache.kyuubi.Logging
+import org.apache.kyuubi.kubernetes.test.MiniKube
 import org.apache.kyuubi.operation.SparkQueryTests
 
-// TODO: [KYUUBI-863] Support test Spark engine using k8s master with minikube
-class KubernetesJDBCTestsSuite extends SparkQueryTests with Logging {
+/**
+ * This test is for Kyuubi Server on Kubernetes with Spark engine:
+ *
+ *   Real World                   Kubernetes Pod (docker container)
```

Comment:
I suggest to remove `(docker container)` in description, because a Pod compose of one or more containers, and K8s has dropped the support of DockerShim

Reply:
yeah we can remove the word